src/main/java/network/brightspots/rcv/GuiConfigController.java

```diff
@@ -613,7 +613,8 @@ private void setTextFieldToInteger(TextField textField, Integer value) {
     textField.setText(value != null ? Integer.toString(value) : "");
   }
 
-  private void setDefaultValues() {
+  // populate widgets with suggested default values for settings which allow them
+  private void setWidgetDefaultValues() {
```

Comment:
Please change to `setGuiDefaultValues()`.

Reply:
How about setUiDefaultValues?

Reply:
I guess? Technically the flags we feed into the CLI count as "UI" too.